include/cucumber-cpp/internal/CukeEngine.hpp

```diff
@@ -59,7 +61,7 @@ class PendingStepException : public InvokeException {
  * It uses standard types (as much as possible) to be easier to call.
  * Returns standard types if possible.
  */
-class CukeEngine {
+class CUCUMBER_CPP_EXPORT CukeEngine {
```

Comment:
AFAIK MSVC should support exporting individual methods and not the entire class. I can test this with `dumpbin /EXPORTS` (tho I'm not on my Windows machine now, so I'll comment again once I've verified that).